components/BalanceTable/index.js

```diff
@@ -29,7 +29,7 @@ const BalanceTable = ({ balances }) => {
                 {formatNumber(balance)}
             </td>
             <td type="num" align="right">
-              {formatDollarAmount(balance * tokens[address].usdPrice)}
+              {tokens[address] && formatDollarAmount(balance * tokens[address].usdPrice)}
```

Comment:
sometimes using an `&&` in jsx will produce unlikely results. 

see https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

its suggested to use ternaries rather then &&.

Reply:
Thanks, fixed it

Reply:
Thanks!